_episodes/15-practices.md

```diff
@@ -22,7 +22,47 @@ no different from drama, music, or athletics.
 And as in those fields,
 we have a collection of small tips and tricks to make teaching work better.
 
-### Challenges
+## The Code of Conduct
+
+Software Carpentry and Data Carpentry have a shared code of conduct
+(online [here]({{ site.swc_site }}/conduct/) and [here]({{ site.dc_site }}/code-of-conduct/))
+and participants in our workshops are required to abide by it.
+Hosts *must* point people at it during registration,
+and instructors *must* remind attendees of it at the start of the workshop.
+The Code of Conduct doesn't just tell everyone what the rules are:
+it tells them that there *are* rules,
+and that they can therefore expect a comfortable learning experience.
+
+What's the purpose of a Code of Conduct?
+:   A Code of Conduct cannot stop people from being offensive, any more
+    than laws against theft stop people from stealing.  What the CoC
+    *can* do is make expectations and consequences clear.
+
+Do workshop participants ever actually violate the Code of Conduct?
+:   Very rarely, but it has happened.
+
+But what about free speech?
+:   People are free to say what they want, but that doesn't mean they
+    are free to say it in our workshops---particularly not if it is
```

Comment:
Having engaged with folks on this kind of thing ('free speech') before, I'll just say that many won't believe that the workshops are a special case. Given the variance accross nations about how this kind of thing stands legally, I'm not sure it's actually worth explaining that workshops aren't usually held on public property. But, it may be worth drawing an analogy with a classroom. I don't think anyone in any nation would argue that a professor has the prerogative to eject a disruptive student from their classroom, even if that student has paid to be there.